cmd/sebak/cmd/node.go

```diff
@@ -188,6 +188,8 @@ func parseFlagsNode() {
 		}
 	}
 
+	logHandler = logging.CallerFileHandler(logHandler)
```

Comment:
Looks like you always overwrite the loghandler. Is that really what we want ? I would have just changed line 181.

Reply:
I misread the code. LGTM!

Reply:
Yep! Thanks! `:-)`